src/api/decoding-scanned-codes.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+parent: API Reference
+title: Decoding Scanned Codes
+permalink: /api/decoding-scanned-codes
+---
+
+# Decoding Scanned Codes
+
+A scanned code can be a either QR code or a barcode that a merchant or a patron scans.
+The scanned code can be decoded to check the provider of the code.
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Scanned Code
+
+{% h4 Mandatory Fields %}
+
+|    Field    |        Type        |                           Description                            |
+| :---------- | :----------------- | :--------------------------------------------------------------- |
+| code        | String             | The code that has been scanned.                                  |
+| scannedBy   | String             | The entity that scanned the code. Can be `merchant` or `patron`. |
```

Comment:
Entity could mean POS. Could you get specific about how this affects the response?

Reply:
I've changed it from entity to person. I'm not sure what you mean by how it affects the response? Could you elaborate?

Reply:
`scannedBy` is a thing you provide right? What types of things would a merchant or patron get back? Should a terminal integrator say they're a merchant? What about a POS? When would you use patron?

Reply:
I've left it as "The person that scanned the code". The only value this field can be is "merchant" so the feedback might not be relevant anymore. Let me know if you still would like me to change it.

Reply:
I've moved this conversation to a thread against the current change so it can be viewed in context.

https://github.com/centrapay/centrapay-docs/pull/443/files#r934050452